Marlin/src/pins/stm32/pins_LONGER3D_LK.h

```diff
@@ -16,28 +16,28 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
+#pragma once
 
 /**
- * ALFAWISE U30 (STM32F103VET6) board pin assignments
+ * Longer3D LK1/LK2 & Alfawise U20/U30 (STM32F103VET6) board pin assignments
  */
 
 #if !defined(__STM32F1__) && !defined(STM32F1xx)
   #error "Oops! Select a STM32F1 board in 'Tools > Board.'"
 #elif HOTENDS > 1 || E_STEPPERS > 1
-  #error "LONGER3D only supports 1 hotend / E-stepper. Comment out this line to continue."
+  #error "Longer3D board only supports 1 hotend / E-stepper. Comment out this line to continue."
 #endif
 
 #define BOARD_NAME "Longer3D"
-#define ALFAWISE_UX0             // Common to all Alfawise STM32F1 boards
+#define ALFAWISE_UX0             // Common to all Longer3D STM32F1 boards (used for Open drain mosfets)
 
+//#define DISABLE_DEBUG          //  We still want to debug with STLINK...
+#define DISABLE_JTAG             //  We free the jtag pins (PA15) but keep STLINK
                                  //  Release PB4 (STEP_X_PIN) from JTAG NRST role.
-//#define DISABLE_DEBUG          //  > Hobi : We still want to debug with STLINK...
-#define DISABLE_JTAG             //  we free the jtag pins (PA15) but keep STLINK
-
 //
 // Limit Switches
 //
-#define X_STOP_PIN         PC1   // pin 16
+#define X_MIN_PIN          PC1   // pin 16
```

Comment:
If all these pins have headers on the board then all these endstop pins can be uncommented.

Reply:
https://aliexpress.com/item/32944556459.html

Reply:
yes, we have 6 endstop connectors

Reply:
but... Zmax can be used for the bltouch in some cases

Reply:
I agree, the unmarked board makes it hard to tell. But the plugs on the left look like they are intended for 6 endstops.

Reply:
Aha, I see you already answered my question.

Reply:
please let them commented please, they are not wired, and E is using XMAX

Reply:
but they are named like that on the board itself...

Reply:
https://drive.google.com/file/d/1-q67OfhRc1eztqRmlJGDw0tPHCFmnhj7/view?usp=sharing